packages/editor/src/lib/editor/types/misc-types.ts

```diff
@@ -55,6 +55,22 @@ export interface TLCameraOptions {
 	constraints?: TLCameraConstraints
 }
 
+/** @public */
+export interface AssetContextProps {
+	screenScale: number
+	steppedScreenScale: number
```

Comment:
i keep going back and forth on whether `screen` in these names is needed/confusing..... hmmmm...........

Reply:
also i vaguely feel like these should be flipped (scale & rawScale, maybe?) for pit-of-success type reasons but don't feel strongly

Reply:
haha yeah, i feel ya on the tough choices on naming here. i could see it going either way myself, too. 🙃